v20.2/performance.md

```diff
@@ -15,44 +15,52 @@ This document is about CockroachDB’s performance on benchmarks. For guidance o
 
 ## Scale
 
-TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors, and CockroachDB can process **1.2M tpmC with 100,000 warehouses, a nearly perfect score.** For a refresher on what exactly TPC-C is and how it is measured, see [Benchmarks used](#benchmarks-used) below.
+TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors, and CockroachDB can process **1.68M tpmC with 140,000 warehouses, a nearly perfect score.** As shown in the chart below, this is a 42% improvement in efficiency over the results from CockroachDB 19.2.
```

Comment:
Please confirm if this is still a "nearly perfect score".  FWIW I'd love to replace this phrase with something more appropriate for technical docs (IMO) like "98% efficiency" or the like.